stable/ark-server/templates/hook-delete-backups.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: delete-ark-backups
+  namespace: {{ .Values.namespace.backup.name }}
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": hook-succeeded
+    "helm.sh/hook-weight": "1"
+spec:
+  template:
+    metadata:
+      name: delete-ark-backups
+      namespace: {{ .Values.namespace.backup.name }}
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: delete-ark-backups
+          image: claranet/gcloud-kubectl-docker
```

Comment:
I'm a bit scared to run a kubectl image from untrusted source, could you add this as a variable

```image: {{ .Values.kubectl.image.repository }}/{{ .Values.kubectl.image.tag }}```

Reply:
I understand, although it's just a simple image with those app installed

Reply:
I agree with @gianrubio. Moreover, as is the image doesn't point to a precise version, and that is against the [best practices](https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/pods.md#images). Lastly I wasn't able to find sources for this image on [docker hub](https://hub.docker.com/r/claranet/gcloud-kubectl-docker/) or on [github](https://github.com/claranet) :)

Reply:
I already changed it in the last commit :)